templates/hydra.yaml.j2

```diff
@@ -18,9 +18,15 @@ serve:
       allowed_origins:
         - "*"
 urls:
+  {%- if consent_url %}
   consent: {{ consent_url }}
+  {%- endif %}
+  {%- if error_url %}
   error: {{ error_url }}
+  {%- endif %}
+  {%- if login_url %}
   login: {{ login_url }}
+  {%- endif %}
```

Comment:
Since we have the default urls set up even if the relation doesn't exist and thus those urls will always have some value provided, we don't need the conditions

Reply:
dismiss this comment if you decide to go with [this](https://github.com/canonical/kratos-operator/pull/49/files#diff-47cd73aae729ed107a6f43de89a621636ce35f09aa20c357b362a10d8b5f848bR14) approach